.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Snuck this one in, but I think not many people use it, so I'm ok with removing it as well